resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -328,6 +348,12 @@ class VisualEffect
                 }
                 Set-ItemProperty -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty) -Value $transparencyValue 
             }
+            if ($null -ne $this.MessageDurationInSeconds) 
+            {
+                $messageDurationValue = if ($this.MessageDurationInSeconds -lt 5) { 5 } elseif ($this.MessageDurationInSeconds -gt 300)  { 300 } else {$this.MessageDurationInSeconds}

```

Comment:
I would first check if the message duration is between 5 and 300 and throw and error message if the check is not satisfied. Here is an example of how you would do that: https://github.com/microsoft/winget-dsc/blob/edbba7796fd007c33cd4f9be677dfa46bb6d9efa/resources/Microsoft.Windows.Developer/Microsoft.Windows.Developer.psm1#L97-L100